conduit/telemetry/README.md

```diff
@@ -0,0 +1,16 @@
+# Conduit Telemetry
+
+The goal is to gain information about how our users use Conduit in the wild. This currently does not include things like logging (logrus) or metrics (Prometheus and Datadog for internal performance tests).
+
+We want to closely follow how algod reports telemetry. However, there is some divergence in the client, e.g. currently using ElasticSearch, but in process of migrating to OpenSearch. We currently use the official [Go opensearch client](https://opensearch.org/docs/latest/clients/go/).
```

Comment:
I'd suggest leaving out the comment about algod. Maybe just mention that the structure is similar to algod and that we're using opensearch, that way it wont become outdated once algod migrates.

Reply:
Agreed, deleted this section [30adcc3](https://github.com/algorand/conduit/pull/74/commits/30adcc329f7bc80046c681c1fdec7f31509d6f7d)